samples/subsys/zbus/benchmark/src/benchmark.c

```diff
@@ -197,18 +197,21 @@ static void producer_thread(void)
 
 		zbus_chan_notify(&bm_channel, K_MSEC(200));
 	}
-	uint32_t duration = (k_uptime_get_32() - start);
+	uint32_t end = sys_clock_cycle_get_32();
+	uint32_t duration = k_cyc_to_ns_near32(end) - k_cyc_to_ns_near32(start);
 
 	if (duration == 0) {
 		LOG_ERR("Something wrong. Duration is zero!\n");
+		LOG_ERR("start = %" PRIu32 "\n", start);
+		LOG_ERR("end = %" PRIu32 "\n", end);
```

Comment:
These two lines (205 and 206) are unnecessary. Line 204 is telling that already.

Reply:
Done!